### PR TITLE
SCP 2536 Update documentation for plutus core executables

### PR DESCRIPTION
**Pull request by thealmarty**
Fixed some typos and updated documentation.

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

Pre-merge checklist:
- [ ] Someone approved it
- [ ] Commits have useful messages
- [ ] Review clarifications made it into the code
- [ ] History is moderately tidy; or going to squash-merge

### CODE REVIEW

File: plutus-core/README.md [hunk 1]
```diff
@@ -72,6 +74,7 @@ Run `nix build -f default.nix plutus.haskell.packages.plutus-core.components.exe
 
 Run `stack install plutus-core` in your terminal being in any subfolder of `plutus`. Once the build finishes, you'll be shown the following lines:
 
+<!-- @thealmarty can't successfully install plutus-core, can someone else verify/update this? -->
```
Comment (thealmarty) [inline plutus-core/README.md:77]: This needs to be removed before merging. Hopefully someone can confirm this.

**Comment by thealmarty** [comment]
Not sure why CI is failing.

File: plutus-core/README.md (hunk 1 above)
Comment (michaelpj) [reply plutus-core/README.md:77]: We shouldn't recommend that people use `install`-based workflows. Better to use `cabal run` or something.

**Review by michaelpj — APPROVED** [review]
CI is having trouble, let me know when you're happy with this and I'll force-merge it!

File: plutus-core/README.md (hunk 1 above)
Comment (thealmarty) [reply plutus-core/README.md]: Great! Updated. Is the "Tests" section under it outdated? I can't seem to find the generator?

**Comment by thealmarty** [comment]
> CI is having trouble, let me know when you're happy with this and I'll force-merge it!

Just waiting to see if I should delete the "Tests" section in the README. Otherwise, if you're happy it's ready to merge.

**Comment by michaelpj** [comment]
I think you can bin it - it was originally written for some external people we're not working with any more.

**Comment by thealmarty** [comment]
> I think you can bin it - it was originally written for some external people we're not working with any more.

Done. Ready to merge.

**Pull request merged**